### PR TITLE
[refactor] endpoint params validator

### PR DESCRIPTION
**Pull request by dev-jotape**
## Changes
Refactor endpoint validators to validate the request params.

### CODE REVIEW

File: packages/api/src/controllers/v2/community/details.ts [hunk 1]
```diff
@@ -16,30 +16,18 @@ class CommunityController {
     getManagers = (req: RequestWithUser, res: Response) => {
```
Comment (obernardovieira) [inline packages/api/src/controllers/v2/community/details.ts:16]: We can do something like https://github.com/impactMarket/backend/blob/main/packages/api/src/controllers/v2/microcredit/list.ts#L22, so we avoid the casts like `orderBy as string` or  `lastActivity_lt as string` below.

File: packages/api/src/controllers/v2/community/details.ts [hunk 2]
```diff
@@ -57,48 +45,31 @@ class CommunityController {
...
-        }
+        const { state, offset, limit, orderBy } = req.query;
         this.detailsService
             .listBeneficiaries(
                 req.user.address,
                 parseInt(req.params.id, 10),
-                parseInt(offset, 10),
-                parseInt(limit, 10),
+                offset ? parseInt(offset as string, 10) : config.defaultOffset,
+                limit ? parseInt(limit as string, 10) : config.defaultLimit,
```
[the remark this answers is not in the document]
Comment (obernardovieira) [reply packages/api/src/controllers/v2/community/details.ts:54]: Also, be careful with where are the config is fetched from. I's being fetched from core, but in this case, it should be from the api

**Review by obernardovieira — CHANGES_REQUESTED** [review]

File: packages/api/src/controllers/v2/community/details.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (dev-jotape) [reply packages/api/src/controllers/v2/community/details.ts]: Nice! I changed to follow this pattern

File: packages/api/src/controllers/v2/community/details.ts (hunk 1 above)
Comment (dev-jotape) [reply packages/api/src/controllers/v2/community/details.ts]: done!

File: packages/api/src/validators/community.ts [hunk 3]
```diff
@@ -1,7 +1,38 @@
 import { Joi, celebrate } from 'celebrate';
 
+import { ContainerTypes, ValidatedRequestSchema } from '~utils/queryValidator';
+import { config } from '@impactmarket/core';
```
Comment (obernardovieira) [inline packages/api/src/validators/community.ts:4]: The config here is used for `defaultLimit` and `defaultOffset`. This are api-specific, so avoid using the core config.

File: packages/api/src/validators/story.ts [hunk 4]
```diff
@@ -1,18 +1,28 @@
 import { Joi, celebrate } from 'celebrate';
 
-import { createValidator } from '../utils/queryValidator';
+import { ContainerTypes, ValidatedRequestSchema, createValidator } from '../utils/queryValidator';
+import { config } from '@impactmarket/core';
```
Comment (obernardovieira) [inline packages/api/src/validators/story.ts:4]: Same about the config.

**Pull request merged**